@navikt/core/react/src/button/button.stories.tsx

```diff
@@ -187,6 +187,6 @@ export const Chromatic: Story = {
 export const ChromaticForcedColors: Story = {
   render: Chromatic.render,
   parameters: {
-    chromatic: { disable: false, forcedColors: "active" },
+    chromatic: { disable: false },
```

Comment:
Please don't merge this change (at least not yet). I have contacted Chromatic about this bug, so need it to be reproducible 🙂